pkg/cephconfig/config_test.go

```diff
@@ -0,0 +1,580 @@
+package cephconfig
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+	"testing"
+
+	pb "github.com/clyso/ceph-api/api/gen/grpc/go"
+)
+
+func TestConfig_Search_FilteringAndSorting(t *testing.T) {
+	ctx := context.Background()
+	cfg, err := NewConfig(ctx, nil, true) // skipUpdate: true for testing
+	if err != nil {
```

Comment:
consider using testify library for asserts. it reduces amount of boilerplate. see https://github.com/clyso/ceph-api/blob/c7ac5d99e1c5e871084aef0e5d52412b218a6333/test/auth_test.go#L28-L30